  camera_imu_calibration: {
  xHat0: [0.5, 0.5, -0.5, 0.5,
          0, 0 ,2,
          0, 0, 0,
          0, 0, 0,
          0, 0, 0,
          0.01, -0.18, -0.005,
          0.5, 0.5, 0.5, 0.5],

  P0: [0.01, 0.01, 0.01, #Attitude
       0.2, 0.2, 0.2,    #Position
       0.1, 0.1, 0.1, #Velocity
       0.01, 0.01, 0.01,  #Bias Gyro
       0.01, 0.01, 0.01,  #Bias Accel
       0.05, 0.05, 0.05,     #Arm
       0.02, 0.02, 0.02],  #Offset Attitude

  #     GYRO                            ACCEL                           GYRO BIAS                   ACCEL BIAS
  Q: [0.01, 0.01, 0.01,                  0.5, 0.5, 0.5,             0.000001, 0.000001, 0.000001,          0.00001, 0.00001, 0.00001],
#  Pixel noise STANDARD DEVIATION
  R: [5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5],


  beta: [0.95, 0.95, 0.95, #attitude
         0.95, 0.95, 0.95 , #position
         1, 1, 1,         #velocity
         0.5, 0.5, 0.5, #gyro bias
         0.5, 0.5, 0.5, #accel bias
         0.01, 0.01, 0.01, #pc_i
         0.001, 0.001, 0.001] #attitude offset
}